composer/workflows/dataflow_template_operator_tutorial/create_dataset_and_table_test.py

```diff
@@ -0,0 +1,67 @@
+from google.cloud import bigquery
+import pytest
+import google.cloud.exceptions
+import uuid
+
+client = bigquery.Client()
+dataset_UUID = str(uuid.uuid4()).split("-")[0]
+
+
+@pytest.fixture(scope="module")
+def dataset():
+  
+    dataset_id = format(client.project)+".sample_dataset"+dataset_UUID
```

Comment:
How about the following? I'd use `str.format()`. I'd also add separator between the `sample_dataset` and the uuid part.

```
"{}.sample_dataset_{}".format(client.project, dataset_UUID) 
```

Also, if you put the variable in global scope, you can re-use it in the test too.

Reply:
separator comment valid, but f strings are now preferred over str.format in Python land :)